### ISSUE TITLE
Execute Temportal Queries

### ISSUE THREAD

**Issue by vertonghenb** [findulov/EntityFrameworkCore.TemporalTables#6]
I couldn't find any docs on how to fire some temportal queries using linq are there any samples or is this not supported?

**Comment by findulov** [comment]
Hi, at this point this is not supported but I've definitely planned it for future releases.
The main goal is to support temporal tables creation/deletion (current functionality) and ability to query temporal data.

**Comment by lpperras** [comment]
Might not be as complete as what you would like but I just created a PR to query the temporal tables to get entities at specified dates.
[https://github.com/findulov/EntityFrameworkCore.TemporalTables/pull/10](url)

**Comment by KarlJorgen** [comment]
Thanks for the AsOf implementation.
When I use Include the related data is not loaded as of the specified date. Only the main entity
Any generic method to get that at the moment?

**Comment by findulov** [comment]
No, sorry to inform you that at this point it's not possible to fetch related data but I keep thinking of an appropriate way to implement that behavior.

**Comment by MikeOtown** [comment]
Any progress on related tables?  What is the current functionality?  .AsOf() only works if there is a single table in the query?
